src/ray/rpc/client_call.h

```diff
@@ -258,6 +274,12 @@ class ClientCallManager {
     return call;
   }
 
+  // For testing purposes only.
+  void StampContext(grpc::ClientContext &context) {
```

Comment:
Try to move to test file.
We can have a getter function to retrieve cluster_id in GcsClient